### PR TITLE
(MAINT) merge down from master for 2.6.0 release

### PR DESCRIPTION
**Pull request by cprice404**

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -1,8 +1,8 @@
 (def clj-version "1.7.0")
-(def tk-version "1.4.0")
-(def tk-jetty-version "1.5.9")
+(def tk-version "1.5.0")
+(def tk-jetty-version "1.5.10")
 (def ks-version "1.3.1")
-(def ps-version "2.5.1-stable-SNAPSHOT")
+(def ps-version "2.6.0-master-SNAPSHOT")
```
Comment (camlow325) [inline project.clj]: Should we change this back to 'stable' as we merge down?

Comment (cprice404) [inline project.clj]: d'oh, yeah.

**Comment by camlow325** [comment]
I did a git diff of this PR against the current HEAD of master.  Looked identical to me.  So 👍 from me.

**Comment by cprice404** [comment]
I changed it to `stable`.

**Comment by camlow325** [comment]
Corresponding commit passed through Jenkins smoke tests on the master branch so I think this is good to come in.  Merging...

**Pull request merged**